### PR TITLE
Load location_id instead of all args in controls.py

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: solar/solar/events/controls.py [hunk 1]
```diff
@@ -97,14 +98,7 @@ def insert(self, changed_resources, changes_graph):
         if self.parent_node in changes_graph:
             if self.child_node not in changes_graph:
                 # TODO: solve this circular import problem
```
Comment (loles) [inline solar/solar/events/controls.py]: remove this todo

Comment (loles) [inline solar/solar/events/controls.py]: remove this todo

File: solar/solar/events/controls.py [hunk 2]
```diff
@@ -122,15 +116,8 @@ class StateChange(Event):
...
-        from solar.core import resource
-        try:
-            loaded_resource = resource.load(self.parent)
-        except KeyError:
-            # orm throws this error when we're NOT using resource there
-            location_id = ''
-        else:
-            location_id = loaded_resource.args['location_id']
+
+        location_id = Resource.get(self.parent).inputs._get_field_val('location_id')
```
Comment (pigmej) [inline solar/solar/events/controls.py]: `Resoruce.get(self.parent).inputs['location_id']` :)

File: solar/solar/events/controls.py [hunk 3]
```diff
@@ -97,14 +98,7 @@ def insert(self, changed_resources, changes_graph):
...
                 # TODO: solve this circular import problem
-                from solar.core import resource
-                try:
-                    loaded_resource = resource.load(self.child)
-                except KeyError:
-                    # orm throws this error when we're NOT using resource there
-                    location_id = ''
-                else:
-                    location_id = loaded_resource.args['location_id']
+                location_id = Resource.get(self.child).inputs._get_field_val('location_id')
```
Comment (pigmej) [inline solar/solar/events/controls.py]: same there:
`Resoruce.get(self.parent).inputs['location_id']`

**Pull request merged**